languagetool-dev/src/main/java/org/languagetool/dev/diff/LightRuleMatchParser.java

```diff
@@ -54,18 +55,17 @@
     ObjectMapper mapper = new ObjectMapper();
     List<LightRuleMatch> ruleMatches = new ArrayList<>();
     int lineCount = 1;
-    try (Scanner scanner = new Scanner(inputFile)) {
-      while (scanner.hasNextLine()) {
-        String line = scanner.nextLine();
+    for (String line : Files.readAllLines(inputFile.toPath())) {
```

Comment:
This should also use scanner to avoid reading everything into RAM.

Reply:
Done, and I removed the try/catch, because no exception is thrown from the enclosed methods.